p2p/chain.py

```diff
@@ -423,10 +423,17 @@ def __init__(self,
             'receipt')
         self.logger.debug("Got block receipts for chain segment")
 
-        # FIXME: Get the bodies returned by self._download_block_parts above and use persit_block
-        # here.
         for header in headers:
-            await self.wait(self.db.coro_persist_header(header))
+            uncles: Tuple[BlockHeader, ...] = tuple()
```

Comment:
*nitpick* I'd rather this be in an `else` block of the following `if` statement than to overwrite the local variable.